incubator/hnc/Dockerfile

```diff
@@ -18,9 +18,12 @@ COPY third_party/ third_party/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager ./cmd/manager/main.go
 
+# Copied from kubebuilder scaffold to run as nonroot at
```

Comment:
Maybe: "Following the latest kubebuilder recommendation to use nonroot"? If we re-used kubebuilder today, we would never have used "static:latest"